vector/src/test/java/im/vector/app/features/onboarding/ftueauth/MatrixOrgMissingRegistrationStagesComparatorTest.kt

```diff
@@ -25,7 +25,7 @@ import im.vector.app.test.fixtures.anOtherStage
 import org.amshove.kluent.shouldBeEqualTo
 import org.junit.Test
 
-class FtueMissingRegistrationStagesComparatorTest {
+class MatrixOrgMissingRegistrationStagesComparatorTest {
```

Comment:
Same remark about the name here then

Reply:
https://github.com/vector-im/element-android/pull/6065/commits/1ec99ee89e85ef16823991fd6e00669a84a6131c